dotCMS/src/main/java/com/dotcms/contenttype/business/UniqueFieldAPIHelper.java

```diff
@@ -0,0 +1,71 @@
+package com.dotcms.contenttype.business;
+
+import com.dotcms.business.WrapInTransaction;
+import com.dotcms.content.elasticsearch.business.ESContentletAPIImpl;
+import com.dotcms.util.CollectionsUtils;
+import com.dotmarketing.business.FactoryLocator;
+import com.dotmarketing.exception.DotDataException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This Helper allow you to interact with the unique_fields table
+ */
+public class UniqueFieldAPIHelper {
```

Comment:
I think helper is tied to Resources on dotCMS, for biz logic we used to use Util

Reply:
done https://github.com/dotCMS/core/pull/30332/commits/bc8ee9ff5ad2bf50a2420677772be1feb108f760#diff-949f80e4b1c8d8fcb0be5b680a9e6ec28d60dbaf50ea9e986a6addcdffb7a7b1R16